src/py/flwr/client/app.py

```diff
@@ -160,6 +162,12 @@ class `flwr.client.NumPyClient`.
     # Wrap the NumPyClient
     flower_client = NumPyClientWrapper(client)
 
+    # Remove NumPyClientWrapper.get_properties with the base class version
+    # Client.get_properties if method was not overridden by user-provided
+    # NumPyClient instance
```

Comment:
Can you elaborate this further or rephrase. I don't think I understand whats happening here.

Reply:
@tanertopal updated the comment, is it better now?

Reply:
Yes, thx!